packages/forms/resources/views/components/date-time-picker.blade.php

```diff
@@ -19,6 +19,7 @@
             shouldCloseOnDateSelection: @js($shouldCloseOnDateSelection()),
             state: $wire.{{ $applyStateBindingModifiers('entangle(\'' . $getStatePath() . '\')') }},
         })"
+        x-on:keydown.esc="isOpen() && $event.stopPropagation()"
```

Comment:
does this work?

```suggestion
        x-on:keydown.esc.stop="isOpen()"
```

Reply:
Hmm is that even an Alpine API that should work? Let me check.

Reply:
I think so

Reply:
wait no, it should only stop IF it's isOpen. so you're right

Reply:
Doesn't work. The `stop` modifier is always applied then, no matter what's in the event handler.